code/modules/mob/living/carbon/human/species/species_attack.dm

```diff
@@ -245,18 +245,35 @@
 	if(prob(25))
 		target.apply_effect(1, INCINERATE, 0)
 
-/datum/unarmed_attack/claws/vaurca_bulwark
-	attack_verb = list("punched", "clobbered", "lacerated")
-	attack_noun = list("clawed fists")
+/datum/unarmed_attack/vaurca_bulwark
+	attack_verb = list("punched", "pulverized", "hammered")
+	attack_noun = list("fists")
+	desc = "Smash into your opponents with the strength the Queens gave you. Not as sharp as other species' claws, but yours hit a hell of a lot harder"
 	eye_attack_text = "claws"
 	eye_attack_text_victim = "claws"
-	attack_name = "clawed fists"
+	attack_name = "bulwark punch"
+	attack_sound = 'sound/weapons/heavysmash.ogg'
 	shredding = TRUE
 
-	damage = 7.5
+	damage = 10
 	attack_door = 20
 	crowbar_door = TRUE
 
+/datum/unarmed_attack/vaurca_bulwark/apply_effects(var/mob/living/carbon/human/user,var/mob/living/carbon/human/target,var/armor,var/attack_damage,var/zone)
+	..()
+	if(prob(25))
+		playsound(user, 'sound/weapons/push_connect.ogg', 50, 1, -1)
+		user.visible_message("<span class='danger'>[user] shoves hard, sending [target] flying!</span>")
+		step_away(target,user,15)
+		sleep(1)
+		step_away(target,user,15)
+		sleep(1)
+		step_away(target,user,15)
+		sleep(1)
+		step_away(target,user,15)
+		sleep(1)
```

Comment:
Absolutely not.
```suggestion
        var/turf/target_turf = get_ranged_target_turf(target, user.dir, 4)
		target.throw_at(target_turf, 4, 1, user)
```
I have no idea if this code works, but it's leagues closer to what you'd need here.

Reply:
Upon further inspection it looks like this was copied from existing attacks in the code. That's even more horrifying.

Reply:
yeah, i just copied the hk ipc punch, which is a thing i imagine no one has touched in a while